network_simulator/plot_tool.py

```diff
@@ -0,0 +1,300 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from statistics import *
+
+class PlotTool(object):
+    """
+    Produces graph of different data structures.
+    """
+    def __init__(self):
+        pass
+
+    def gen_count_interval_list(self, lst):
+        """
+        Computes a tuple list of counts based on a timestamp list.
+
+        :param list lst: list of timestamps to be converted.
+        :return: list
```

Comment:
State that this is a list of (time, count) tuples and describe what the counts would represent.